step.yml

```diff
@@ -15,101 +18,87 @@ host_os_tags:
 project_type_tags:
   - ios
```

Comment:
every project type which can generate ipa, based on the service's site it has sdk for:

```
iOS SDK
Android SDK
Xamarin Component
Cordova / PhoneGap
Unity Plugin
Adobe Air Extension
Titanium Module
React Native Component
```